plugins/cost-insights/src/components/CostOverviewCard/CostOverviewBreakdownChart.tsx

```diff
@@ -187,7 +188,7 @@ export const CostOverviewBreakdownChart = ({
     const dateTitle = date.toUTC().toFormat(DEFAULT_DATE_FORMAT);
     const items = payload.map((p, i) => ({
       label: p.dataKey as string,
-      value: formatGraphValue(Number(p.value), i),
+      value: formatGraphValue(baseCurrency)(Number(p.value), i),
```

Comment:
Since this is essentially a curried function. Could you not just create the graph value formatter for the specific currency once and re-use it? I think it would have slightly less object allocations and probably more readable.